vendor/assets/stylesheets/dvl/components/navbar.scss

```diff
@@ -1,4 +1,4 @@
-$navBackground: #0074D9 !default;
+$navBackground: #0f1c3c !default;
```

Comment:
We're usually overriding this in the host app, but :+1: on changing the default.